test/jdk/javax/imageio/CachePremissionsTest/CachePermissionsTest.java

```diff
@@ -26,40 +26,15 @@
  * @bug     6684104
  * @summary Test verifies that ImageIO checks all permissions required for
  *           the file cache usage:
- *
  *          no policy file: No security restrictions.
  *             Expected result: ImageIO creates file-cached stream.
- *
- *          w.policy: the case when we have read and write permissions
- *              for java.io.temp directory but have only write permission
- *              for a temp file.
- *             Expected result: ImageIO create a memory-cached stream
- *              image output stream.
- *
- *          rw.policy: the case when we  have read and write permissions
- *              for java.io.temp directory but have only read and write
- *              permission for a temp cache file.
- *             Expected result: ImageIO creates a memory-cached stream
- *              because temporary cache file can not be deleted.
- *
- *          rwd.policy: the case when we  have read and write permissions
- *              for java.io.temp directory and have all required permissions
- *             (read, write, and delete) for a temporary cache file.
- *             Expected result: ImageIO creates file-cached stream.
- *
- *           -Djava.security.debug=access can be used to verify file permissions.
- *
  * @run     main CachePermissionsTest true
- * @run     main/othervm -Djava.security.manager=allow CachePermissionsTest false w.policy
- * @run     main/othervm -Djava.security.manager=allow CachePermissionsTest false rw.policy
- * @run     main/othervm -Djava.security.manager=allow CachePermissionsTest true rwd.policy
```

Comment:
Looks to me like we should delete these 3 policy files.
Also this test isn't about Permissions anymore. So should be renamed to CacheUsedTest 
Then we can get rid of the misspelt directory. in a follow up.Probably do this